controllers/vrg_volrep_test.go

```diff
@@ -1835,7 +1835,7 @@ func pvcClusterDataProtectedStatusVerify(
 
 type pvcPostDeleteVerify func(types.NamespacedName, string)
 
-func vrAndPvcDeletionTimestampsRecentVerify(pvcNamespacedName types.NamespacedName, pvName string) {
+func vrAndPvcDeletionTimestampsRecentVerify(pvcNamespacedName types.NamespacedName, _ string) {
```

Comment:
This too

Reply:
This function must have signature:
type pvcPreDeleteVerify func(ramendrv1alpha1.VolumeReplicationGroup, types.NamespacedName, string)